### PR TITLE
Fix for linustechtips.com "Author" badge

### PR DESCRIPTION
**Pull request by busybox11**

### CODE REVIEW

**Review by Gusted — COMMENTED** [review]
Where can I find such element?

**Comment by busybox11** [comment]
Basically on any forum post where there is an author card. I believe you'll be able to find it on the first message of every post, can't verify it rn though, sorry

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7153,6 +7153,15 @@ INVERT
 
 ================================
 
+linustechtips.com
+
+CSS
+.ipsBadge, a.ipsBadge {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7159]:
```suggestion
.ipsBadge,
a.ipsBadge {
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -7153,6 +7153,15 @@ INVERT
...
+    background: #af340450 !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7160]:
```suggestion
    --darkreader-bg--badge--background: ${white} linear-gradient(rgba(0,0,0,0.9), rgba(0,0,0,0.9))
```

**Review by Gusted — APPROVED** [review]

**Pull request merged**